### ISSUE TITLE
bug-hunter-file output

### ISSUE THREAD

**Issue by boudiccas** [Kungsgeten/yankpad#17]
my emacs is failing to load properly and according to bug-hunter-file, the problem lies in 
''''
Doing some initial tests...
Initial tests done. Hunting for the cause...
"/home/boudiccas/.emacs.d/elpa/yankpad-20170220.55/yankpad-autoloads.el", line 4 pos 0:
  The following error was signaled here:
    (wrong-type-argument stringp nil)
  Caused by the following expression:
    (add-to-list 'load-path
                 (directory-file-name
                  (or
                   (file-name-directory nil)
                   (car load-path))))
''''

This is how its loaded in my config -
''''
(use-package yankpad
  :ensure t
  :defer 10
  :init
  (setq yankpad-file "/home/boudiccas/.emacs.d/org/yankpad.org")
  :config
  (bind-key "C-f1" 'yankpad-map)
  (bind-key "C-f2" 'yankpad-expand)
  ;; If you want to complete snippets using company-mode
  (add-to-list 'company-backends #'company-yankpad))
''''

**Comment by Kungsgeten** [comment]
Hmm, that's strange. I'm not sure what could cause that problem.

**Comment by E3D3** [comment]
same here. The yankpad-map function seems to be unknown. Evaluatings its definition in yankpad.el removed the error. I'm not good in Elisp but hopes this helps a little

**Comment by Kungsgeten** [comment]
Hmm, ok. I've added an autoload cookie for `yankpad-map`, please let me know if that helps. It will take a couple of hours before the new version is on MELPA though.

**Comment by E3D3** [comment]
ehh, after reading your last comment I discovered that adding a (require 'yankpad) solved it, something I overlooked because I don't use 'use-package'. Guess that this still won't help 'boudiccas', althougth in both our cases seems to be yankpad-map the only missing function. After updating the new yankpad, I see that even the (require 'yankpad) not necessary anymore is. Sorry for my confussion & Thanks for your quick respons, and of course for sharing Yankpad.
